adapters/improvedigital/improvedigital.go

```diff
@@ -78,3 +100,26 @@ func NewImprovedigitalBidder(endpoint string) *ImprovedigitalAdapter {
 		endpoint: endpoint,
 	}
 }
+
+func getMediaTypeForImp(impID string, imps []openrtb.Imp) (openrtb_ext.BidType, error) {
+	for _, imp := range imps {
```

Comment:
How well does this code work for multiformat imps? For example, if given a multiformat `imp` that includes both banner and video, is your bidder guaranteed to return a banner type bid?

Reply:
Our ADServer don`t support multiformat, only banner will be used ignoring video/native/etc. So adapter have the some logic as server to prefer banner by default. Agree, case is good but right now not expected in real life and will make nobid (204 status) with alarm to check publisher site configuration